### PR TITLE
Pouta vm creating and connecting review

### PR DESCRIPTION
**Pull request by lvarin**
Two pages were reviewed, the "launch VM" and the "connect to VM".  Mostly to separate and itemize the current content
    
    PowerShell for windows added
    Putty information updated
    Steps are separated and clarified
    Tips and warnings now are in clear separate boxes
    Centos8 user fixed
    Miscellaneous image editing

### CODE REVIEW

**Comment by lvarin** [comment]
https://csc-guide-preview.rahtiapp.fi/origin/pouta-ssh-improve/

File: docs/cloud/pouta/connecting-to-vm.md [hunk 1]
```diff
@@ -1,142 +1,176 @@
...
+Host <machine_name>
+Hostname <floating ip>
+User <user_name>
+IdentityFile <private_key_with_path>
+```
 
-Now you should be able to connect to the public floating IP of your VM 
-in cPouta (or VM local IP in case of ePouta) using SSH without the need 
-of specifying the key:
+* In **Host** you should write the machine name (to later use to connect using `ssh machine_name`.
```
Comment (matteopozza) [inline docs/cloud/pouta/connecting-to-vm.md:73]: closing round bracket missing

File: docs/cloud/pouta/connecting-to-vm.md [hunk 2]
```diff
@@ -1,142 +1,176 @@
...
+* In **Hostname** write the floating of the Virtual Machine.
 
-    #for ePouta VMs
-    ssh cloud-user@vm-ip
+* In **User** the user name must be specified.
 
-!!! Tip
-    You can enable *agent forwarding* when connecting through SSH to a 
-    virtual machine by using the *-A* flag.
+* In **IdentityFile** the full apth to the private key must be written, for example `IdentityFile ~/.ssh/id_rsa`.
```
Comment (matteopozza) [inline docs/cloud/pouta/connecting-to-vm.md:79]: path

File: docs/cloud/pouta/connecting-to-vm.md [hunk 3]
```diff
@@ -1,142 +1,176 @@
...
+In order to be able to use the console, **you need to set up a password-based user account first**:
 
-You can open a console session by clicking **Console** in the
-instance dropdown menu:
+* Connected through SSH to your Virtual Machine instance
+* You can use [useradd](https://linux.die.net/man/8/useradd) and or [passwd](https://linux.die.net/man/1/passwd) to set up the account. 
+* As indicated in our [security guidelines](../security/#disable-password-login-use-keys), please **do not enable remote login** for this **password-based account**, but rather use it only in case you need to access the instance though the console.
 
-![Open a console in the web GUI](/img/console-button-horizon.png)
+Once there is a passord based account, with no remote login allowed.
```
Comment (matteopozza) [inline docs/cloud/pouta/connecting-to-vm.md:161]: password

File: docs/cloud/pouta/launch-vm-from-web-gui.md [hunk 4]
```diff
@@ -32,86 +32,142 @@ There are two options for logging in the cPouta's Dashboard: CSC username and pa
...
 
 Before starting your first virtual machine in cPouta/ePouta, you must first set up a SSH key pair and modify the security settings so that you will be able to connect to your virtual machine.
 
+### Selecting the CSC project
+
+![Pouta project selection](../../img/pouta_project_selection.png){ align=left }
+
+You may have more than one CSC project with access to Pouta. You can check this from [my.csc.fi](https://my.csc.fi){:target="_blank"}, where you will be able to see all the projects you have access and which ones have cPouta (or ePouta) activated as a service.
+
+Back in Poutai's interface, make sure that you select the correct project. There are two condiderations here:
```
Comment (matteopozza) [inline docs/cloud/pouta/launch-vm-from-web-gui.md:53]: Pouta

File: docs/cloud/pouta/launch-vm-from-web-gui.md [hunk 5]
```diff
@@ -32,86 +32,142 @@ There are two options for logging in the cPouta's Dashboard: CSC username and pa
...
+
+* A project is a sandbox which contains resources like Virtual Machines and networks, and anyone with access to that project will be able to see and administer all these resources. They may not be able to access a Virtual Machine, as this is determinated by the SSH keys configured in the machine, but they will be able to **delete**, **reboot**, ... etc.
+* Projects are used to determinate billing. Make sure that the costs will go the correct billing project.
```
Comment (matteopozza) [inline docs/cloud/pouta/launch-vm-from-web-gui.md:56]: perhaps "Make sure that the costs will go *to* the correct billing project"?

File: docs/cloud/pouta/launch-vm-from-web-gui.md [hunk 6]
```diff
@@ -32,86 +32,142 @@ There are two options for logging in the cPouta's Dashboard: CSC username and pa
...
-![The Access & Security subpage in the cPouta web interface](/img/pouta-user-guide-keypairs.png 'ssh key pairs')
+1. Give your key a name and click in **Create Key Pair**. You will get a "_keyname.pem_" to save. Save it in your home directory. This will be the last time you will be able to download this **private key**, Pouta does not keep a copy in its servers.
 
-**Figure** The _Access & Security_ subpage in the cPouta web interface
+    ![Create key](../../img/pouta-create-key.png)
 
-Finalize the key installation in the **Linux** and **Mac OS X** environments:
-If you are using Chrome browser in Mac OS X Monterey, you will get keyname.cer instead of keyname.pem.
-The following procedure will remain same.
+    **Figure** The Create Key Pait dialog
```
Comment (matteopozza) [inline docs/cloud/pouta/launch-vm-from-web-gui.md:80]: Pait -> Pair

File: docs/cloud/pouta/launch-vm-from-web-gui.md [hunk 7]
```diff
@@ -32,86 +32,142 @@ There are two options for logging in the cPouta's Dashboard: CSC username and pa
...
+
+#### Linux and Mac
+
+In order to install the key you downloaded in the repvious step (_keyname.pem_ or _keyname.cer_), you must run this commmands:
```
Comment (matteopozza) [inline docs/cloud/pouta/launch-vm-from-web-gui.md:84]: previous

File: docs/cloud/pouta/launch-vm-from-web-gui.md [hunk 8]
```diff
@@ -32,86 +32,142 @@ There are two options for logging in the cPouta's Dashboard: CSC username and pa
...
+A security group can be edited or created in any moment of the Virtual Machine life-cycle. Any change applied to a security group assigned to a Virtual Machine, will be applied instantly to the Virtual Machine.
 
-The "Default" security group comes with rules that allow communication between virtual machines that are all connected to the default security group. If you want your virtual machines to communicate with each other, you need to either add the default security group for them or add similar rules to other security groups used by your virtual machines.
+!!! warning "Do not edit the default security group"
+    As a good practice, we discourage changing the "Default" security group. We recommend instead that you create specific security groups for specific purposes and name them accordinly. For example create a security group called "SSH-VPN" to allow computers from the VPN to SSH/22 to the machines on that security group.
```
Comment (matteopozza) [inline docs/cloud/pouta/launch-vm-from-web-gui.md:152]: accordingly

File: docs/cloud/pouta/launch-vm-from-web-gui.md [hunk 9]
```diff
@@ -32,86 +32,142 @@ There are two options for logging in the cPouta's Dashboard: CSC username and pa
...
-If you do not know your IP address, you can use [https://apps.csc.fi/myip](https://apps.csc.fi/myip) to find out what it is.
+    ![Add rule](../../img/pouta-add-rules-secgroup.png)
 
-You can also open ports to all possible IP addresses. In that case, you would use "0.0.0.0/0" for the network:
-```bash
-Direction   IP Protocol   From Port   To port   Source Group   CIDR
-Ingress     TCP           22          22        CIDR           0.0.0.0/0
-```
-Opening up ports this widely is not recommended and not necessary in most cases. Limiting access to a virtual machines to only those networks that actually need to access it is a good security practice.
+    There is a lot customization available, but in this case it is recommended to use the `SSH` rule that only requires one parameter: `CIDR`. The **Classless Inerdomain Routing** or **CIDR** allows you to specify a subnet (`88.44.55.0/24`) or an specific IP (`88.44.55.77/32`).
```
Comment (matteopozza) [inline docs/cloud/pouta/launch-vm-from-web-gui.md:162]: Classless Inter-Domain Routing

File: docs/cloud/pouta/launch-vm-from-web-gui.md [hunk 10]
```diff
@@ -120,59 +176,70 @@ Opening up ports this widely is not recommended and not necessary in most cases.
...
+        The "Boot from image (creates a new volume)" approach creates an additional volume which is billed normally as mentioned on our [pricing](https://research.csc.fi/pricing-of-computing-services) page.
 
-![Floating IP association options](/img/associate-floating-ip-menu.png 'Associate floating IP menu')
 
-**Figure** Floating IP association options
+1. **Image Name**, this decides which Linux distribution to use. You can select the image that fits more your use case. The images provided by Pouta by default are regularly maintained up to date.
 
-Click the plus to create a new IP, select the IP, select your machine under **Instance**, and click **Associate**. It takes some minutes before you are able to see the second, public IP address in the _Instances view_. Once the second IP is shown as well, your machine has a public IP and is accessible on the internet.
+1. Under the **Access & Security** tab, you need to configure two options. First you need to choose the name of the *Key Pair* you have created in the **Preparatory Stepss**. Secondly you need to select under the **Security Groups** the security group previously created.
```
Comment (matteopozza) [inline docs/cloud/pouta/launch-vm-from-web-gui.md:213]: Steps

File: docs/cloud/pouta/connecting-to-vm.md (hunk 1 above)
Comment (lvarin) [reply docs/cloud/pouta/connecting-to-vm.md]: fixed

File: docs/cloud/pouta/connecting-to-vm.md (hunk 2 above)
Comment (lvarin) [reply docs/cloud/pouta/connecting-to-vm.md]: fixed

File: docs/cloud/pouta/connecting-to-vm.md (hunk 3 above)
Comment (lvarin) [reply docs/cloud/pouta/connecting-to-vm.md]: fixed both

File: docs/cloud/pouta/launch-vm-from-web-gui.md (hunk 4 above)
Comment (lvarin) [reply docs/cloud/pouta/launch-vm-from-web-gui.md]: fixed

File: docs/cloud/pouta/launch-vm-from-web-gui.md (hunk 6 above)
Comment (lvarin) [reply docs/cloud/pouta/launch-vm-from-web-gui.md]: fixed

File: docs/cloud/pouta/launch-vm-from-web-gui.md (hunk 5 above)
Comment (lvarin) [reply docs/cloud/pouta/launch-vm-from-web-gui.md]: fixed

File: docs/cloud/pouta/launch-vm-from-web-gui.md (hunk 7 above)
Comment (lvarin) [reply docs/cloud/pouta/launch-vm-from-web-gui.md]: fixed

File: docs/cloud/pouta/launch-vm-from-web-gui.md (hunk 10 above)
Comment (lvarin) [reply docs/cloud/pouta/launch-vm-from-web-gui.md]: fixed

File: docs/cloud/pouta/launch-vm-from-web-gui.md (hunk 9 above)
Comment (lvarin) [reply docs/cloud/pouta/launch-vm-from-web-gui.md]: fixed

File: docs/cloud/pouta/launch-vm-from-web-gui.md (hunk 8 above)
Comment (lvarin) [reply docs/cloud/pouta/launch-vm-from-web-gui.md]: fixed

**Comment by lvarin** [comment]
All changes have been applied, please check and let me know if there are still things to change

**Review by matteopozza — APPROVED** [review]

**Pull request merged**